### ISSUE TITLE
Submit

### ISSUE THREAD

**Issue by ReinierMaas** [Kipke/InteractieTech#22]
Make sure that the name and student number of the author(s) is (are) included in each deliverable.

Combine all of the deliverables into a single zip file, containing a folder for each deliverable, and submit this zip file using the submit system. If you worked on the assignment with a fellow student, you need to submit the zip file only once for the two of you.

The deadline for this assignment is Monday, March 2, 2015, 12:00 (noon). This deadline is final and will be enforced by the submit system. Do not postpone your submission till the last minute. Submissions by e-mail will not be accepted. After submission your work will be graded.
